app/scenes/ErrorSuspended/ErrorSuspended.js

```diff
@@ -0,0 +1,24 @@
+// @flow
+import React from 'react';
+
+import CenteredContent from 'components/CenteredContent';
+import PageTitle from 'components/PageTitle';
+
+const ErrorSuspended = () => (
+  <CenteredContent>
+    <PageTitle title="Your account has been suspended" />
+    <h1>
+      <span role="img" aria-label="Warning sign">
+        ⚠️
+      </span>{' '}
+      Your account has been suspended
+    </h1>
+
+    <p>
+      Team admin has suspended your account. To re-activate your account, please
```

Comment:
Correct grammar would be "A team admin has suspended your account"

It would be nice if we could actually show them the admins to contact in the future. I've noticed that Slack does this for a number of things.